templates/repo/home.tmpl

```diff
@@ -1,7 +1,20 @@
 {{template "base/head" .}}
+<script id="view-data-script" type="module">
+	let viewFileInfo = window.config.pageData.viewFileInfo || {
+		files:[],
+		fileTreeIsVisible: false,
+		selectedItem: '',
+		apiBaseUrl: {{.RepoLink}},
+	};
+	window.config.pageData.viewFileInfo = viewFileInfo;
+</script>
```

Comment:
I do not think such `window.config.pageData.viewFileInfo` is right.

The code is copied from the DiffFileTree, the diff file tree needs it because it "loads more files".

But I do not see the "ViewFileTree" needs to do so.